src/main/kotlin/hr/foi/pop/backend/request_bodies/ActivateUserRequestBody.kt

```diff
@@ -0,0 +1,4 @@
+package hr.foi.pop.backend.request_bodies
+
+
+data class ActivateUserRequestBody(val activated: Boolean?)
```

Comment:
You know, if you remove the question mark here, Jackson's parsing would break when `activated` attribute would not be appended to the request and all that would be left to do would be to catch a bad JSON format exception and respond with 400.

Reply:
Yeah, meticulous as always. ❤ Tnx!